.github/workflows/windows-ci.yaml

```diff
@@ -42,29 +42,28 @@ jobs:
 
       - name: Build Dependencies
         run: |
-          vcpkg install `@vcpkg.txt --triplet ${{ matrix.cfg.triplet }}
+          vcpkg install "@vcpkg.txt" --triplet ${{ matrix.cfg.triplet }}
```

Comment:
We should (in theory) move to a json manifest that's supported automatically by vcpkg, but that a task for another day

Reply:
Seems we may not have much choice, since vcpkg isn't understanding the response file any more...